### ISSUE TITLE
|| operator parsing error

### ISSUE THREAD

**Issue by yoctoserge** [nginx/njs#588]
Hi! Trying to use external library, via npm. Library has such code line which cannot be parsed (trying with NJS Util):

`for (key in props || {}) {`

Here is the output of the NJS Util:
```
Thrown:
SyntaxError: Unexpected token ")" in /app/main.js:130
make: *** [run-lib] Error 1
```
Is it a bug or somthing not supported intentionally? Is there a way to process the file by webpack, etc, to transpile such constructions?

Thanks!

**Comment by drsm** [comment]
Hi!
@yoctoserge 

Take a look at #351

**Comment by xeioex** [comment]
Fixed as a part of #351.
